TwitchChannelPointsMiner/classes/Twitch.py

```diff
@@ -155,22 +155,22 @@ def get_channel_id(self, streamer_username):
         else:
             return json_response["data"]["user"]["id"]
 
-    def get_followers(self):
-        json_data = copy.deepcopy(GQLOperations.PersonalSections)
-        json_response = self.post_gql_request(json_data)
-        try:
-            if (
-                "data" in json_response
-                and "personalSections" in json_response["data"]
-                and json_response["data"]["personalSections"] != []
-            ):
-                return [
-                    fw["user"]["login"]
-                    for fw in json_response["data"]["personalSections"][0]["items"]
-                    if fw["user"] is not None
-                ]
-        except KeyError:
-            return []
+    def __do_kraken_request(self, query, response_as_json=True):
```

Comment:
While this works now, isn't there a better API? This is a legacy API and will be stopped in the upcoming months (https://dev.twitch.tv/docs/v5).

Seems to be replaced with https://dev.twitch.tv/docs/api/reference#get-users-follows

Reply:
When i want to request helix API on this revision [https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/tree/0bcd1ac0a05815e4baf947e67038d07921a964bf](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/tree/0bcd1ac0a05815e4baf947e67038d07921a964bf), I get 401 error.

![image](https://user-images.githubusercontent.com/43729314/135772680-e8fbb1df-1bad-456d-93d1-d9bea7591238.png)

I guess we need to change the way we authenticate to the API?